docs/builders/linode.mdx

```diff
@@ -120,14 +129,40 @@ Token](https://www.linode.com/docs/platform/api/getting-started-with-the-linode-
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
 source "linode" "example" {
-  image             = "linode/debian9"
+  image             = "linode/debian11"
   image_description = "My Private Image"
   image_label       = "private-image-${local.timestamp}"
   instance_label    = "temporary-linode-${local.timestamp}"
   instance_type     = "g6-nanode-1"
   linode_token      = "YOUR API TOKEN"
   region            = "us-east"
   ssh_username      = "root"
+  private_ip        = true
+
+  authorized_users  = ["your_authorized_username"]
+  authorized_keys   = ["ssh-rsa AAAA_valid_public_ssh_key_123456785== user@their-computer"]
+  stackscript_id    = 1177256
+  stackscript_data  = {
+    "test_key": "test_value_1"
+  }
+
+  interface {
+    purpose       = "public"
+    label         = ""
+    ipam_address  = ""
+  }
+
+  interface {
+    purpose       = "vlan"
+    label         = "vlan-1"
+    ipam_address  = "10.0.0.1/24"
+  }
+
+  interface {
+    purpose       = "vlan"
+    label         = "vlan-2"
+    ipam_address  = "10.0.0.2/24"
+  }
```

Comment:
This might be a bit more accessible if we moved this into a separate "complex" example block.

Reply:
Haha, yep